### ISSUE TITLE
Named Operations handled by AddOperations

### ISSUE THREAD

**Issue by m55624** [gchq/Gaffer#1083]
We need to implement AddOperationsToChain with NamedOperations to ensure the OperationChain within a NamedOperation is also updated.

**Comment by p013570** [comment]
This is related to #1063.

**Comment by p013570** [comment]
The changes have been completed, but it depends on pull request https://github.com/gchq/Gaffer/pull/1092 to be merged in first.

**Comment by p013570** [comment]
# Migration Steps
Named operations are now part of core Gaffer and as long as you have configured a cache service for Gaffer named operations will be available.

This means you should remove the following from your store properties file:
```properties
gaffer.store.operation.declarations=NamedOperationDeclarations.json
```

Remove this dependency:
```xml
<dependency>
    <groupId>uk.gov.gchq.gaffer</groupId>
    <artifactId>named-operation-library</artifactId>		
    <version>${gaffer.version}</version>		
</dependency>
```

Named operations are now resolved via a GraphHook. If you do not manually add the NamedOperationResolver graph hook then it will be automatically added as the first graph hook. So you want to run other graph hooks before the named operations are resolved (such as OperationChainLimiter) then you must manually add the NamedOperationResolver as a graph hook in the required position.

**Comment by p013570** [comment]
Merged into develop-1.0.0.
